Templates/Common-Backend-Scripts/dbbBuild.sh

```diff
@@ -428,7 +436,14 @@ if [ $rc -eq 0 ]; then
   fi
 fi
 
-# Ready to go  TLD: Suggest in the section to echo as much as possible
+
+# Setup build environment and pull external dependencies if an Application Descriptor is found
+if [ $rc -eq 0 ] && [ "$fetchBuildDependencies" == "true" ]; then
+    fetchBuildDependenciesMethod
```

Comment:
If we change to a regular call of the ${fetchBuildDependenciesUtilities} script, it would look cleaner.

Reply:
Agreed. Done